Samples/dynamic-dpi/readme.md

```diff
@@ -41,7 +41,7 @@ The code samples included here will help you with handling DPI changes in your c
   - Office/SharePoint development
   - latest version of Windows SDK
 
-  > **Note:** Some samples may not compile because they might target an older Windows SDK. If this happens you can right-click the solution and choose **Retarget solution**. Then choose the version of the Windows SDK you have installed.
+  > **Note:** Some samples may not compile because they might target an older Windows SDK. If this happens you can select and hold (or right-click) the solution and choose **Retarget solution**. Then choose the version of the Windows SDK you have installed.
```

Comment:
```suggestion
  > **Note:** Some samples may not compile because they might target an older Windows SDK. If this happens you can right-click (or select and hold) the solution and choose **Retarget solution**. Then choose the version of the Windows SDK you have installed.
```